src/transport/net.cc

```diff
@@ -1360,8 +1360,14 @@ static ncclResult_t sendProxyProgress(struct ncclProxyState* proxyState, struct
             // Coverity complains about the size here as pointing to an out-of-scope temporary.  Which is nonsense,
             // since size is a plain integer.
             // coverity[use_invalid:FALSE]
-            NCCLCHECK(proxyState->ncclNet->isend(resources->netSendComm, buff, size, resources->tpRank, sub->sendMhandle, sub, sub->requests+buffSlot));
-            if (sub->requests[buffSlot] != NULL) {
+            void **requestPtr = sub->requests+buffSlot;
+            // for LL/LL128 protocols, completion event for write operation is not needed on the receiver side as
+            // the LL flags are actively polled to detect if full data is received or not, so this hint can be used
```

Comment:
For multi-node, how do you know that the RDMA write is completed without some completion event? The polled flag is set by the CPU proxy thread. 
CC: @nileshnegi

Reply:
are you asking about data sender side or receiver side ? sender side there is always completion event posted by the driver to the app .. receiver side depends on whether WRITE or WRITE_IMM is used by RDMA